### ISSUE TITLE
cp-server images 5.3.x are broken 

### ISSUE THREAD

**Issue by vdesabou** [vdesabou/kafka-docker-playground#1425]
the cp-server docker images 5.3.4 are broken (does not even start) with error:

```
docker run confluentinc/cp-server:5.3.4         
===> ENV Variables ...
/etc/confluent/docker/run: line 29: show_env: command not found
docker run confluentinc/cp-server-connect-base:5.3.4
===> ENV Variables ...
/etc/confluent/docker/run: line 23: show_env: command not found
```

**Comment by vdesabou** [comment]
https://confluentinc.atlassian.net/browse/ST-6539

**Comment by vdesabou** [comment]
🚨 a workaround is implemented, see Implement workaround for #1425 (images broken for 2.3.x) #1644

**Comment by vdesabou** [comment]
it works now with 5.3.7
